src/include/execution/util/hash.h

```diff
@@ -23,7 +23,7 @@ enum class HashMethod : uint8_t { Fnv1, Crc, Murmur2, xxHash3 };
  * for arithmetic values (integers and floats), and generic versions for longer
  * buffers (strings, c-strings, and opaque buffers).
  */
-class Hasher {
+class EXPORT Hasher {
```

Comment:
Why is this needed in this PR?

Reply:
Solved a compilation issue on macOS, I think.